awscli/examples/ssm/add-tags-to-resource.rst

```diff
@@ -1,7 +1,25 @@
-**To add tags to a resource**
+**To add tags to an SSM Document**
 
-This example updates a maintenance window with new tags. There is no output if the command succeeds.
+This example updates an SSM document with new tags. There is no output if the command succeeds.
 
 Command::
 
-   aws ssm add-tags-to-resource --resource-type "MaintenanceWindow" --resource-id "mw-03eb9db42890fb82d" --tags "Key=Stack,Value=Production"
+   aws ssm add-tags-to-resource --resource-type "Document" --resource-id "My-Document" --tags "Key=Quarter,Value=Q3-21"
```

Comment:
It looks like the preferred formatting for examples has changed since you first opened the PR. What do you think about updating the formatting to match the [current formatting style](https://raw.githubusercontent.com/aws/aws-cli/develop/awscli/examples/ssm/add-tags-to-resource.rst) (i.e., using a `\` to break out each parameter/value pair into its own line)?